www/compose.py

```diff
@@ -2,12 +2,12 @@
 
 import sys
 import datetime
-import types
+from __builtin__ import basestring
```

Comment:
I also messed up the print parentheses

Reply:
I think I've used python 2 because asciidoc / a2x use python 2, but fortunately, we don't import any asciidoc module, so it's very appropriate to move forward to python 3!

Reply:
Now that #398 is merged, I'll update this PR. Thanks!